File: src/Api/Model/DB/SituacaoCadastral.cs
using Newtonsoft.Json;
using Realms;
using System;
using System.Collections.Generic;
using System.Text;

namespace CriadorBaseDados.Model.DB
{

    [JsonObject(MemberSerialization.OptIn)]
    public class SituacaoCadastral : RealmObject
    {
        /// <summary>
        /// 2 DIGITOS
        //CÓDIGO DA SITUAÇÃO CADASTRAL
        //01 - NULA
        //02 - ATIVA
        //03 - SUSPENSA
        //04 - INAPTA
        //08 - BAIXADA
        /// </summary>
        [JsonProperty]
        [PrimaryKey]
        public int Cod { get; set; }
        [JsonProperty]
        public string Descricao { get; set; }

    }
}

File: src/Mobile/b2b/lib/model/core/situacao_cadastral.dart
class SituacaoCadastral {
  /// <summary>
  /// 2 DIGITOS
  //CÓDIGO DA SITUAÇÃO CADASTRAL
  //01 - NULA
  //02 - ATIVA
  //03 - SUSPENSA
  //04 - INAPTA
  //08 - BAIXADA
  /// </summary>
  final int cod;
  final String descricao;

  SituacaoCadastral(this.cod, this.descricao);

  SituacaoCadastral.fromJson(Map<String, dynamic> json)
      : cod = json['cod'],
        descricao = json['descricao'];

  Map<String, dynamic> toMap() => <String, dynamic>{
        'cod': cod,
        'descricao': descricao,
      };
}
